### DISCUSSION TITLE
What am I doing wrong with config?

### DISCUSSION THREAD

**Discussion by novakboskov** [decentralized-identity/veramo#1351] [Q&A]
Just following the example from the home page:

```
$ npm i @veramo/cli -g
$ veramo -v
5.6.0

$ mkdir my-agent
$ cd my-agent 
$ veramo config .
$ veramo server
Error creating @veramo/did-provider-key['KeyDIDProvider']: Function.prototype.apply was called on undefined, which is a undefined and not a function. While creating object from pointer: /didManager. While creating object from pointer: /agent. While creating object from pointer: /server
```

It seems like I missed an important part of the docs?

**Comment by mirceanis** [comment]
I don't think you are doing anything wrong.
This was reported once more but I was unable to reproduce it.

Can you try to uninstall veramo, clear npm caches, and then try again?

What version of node/npm are you using?
Did you try also with `yarn` or `pnpm`?
